src/language-server/workspace.ts

```diff
@@ -196,6 +197,9 @@ export class GraphQLWorkspace {
     }
     // If project exists, update the config
     if (project && config) {
+      if (equal(project.config.rawConfig, config.rawConfig)) {
+        return;
+      }
```

Comment:
There are situations where a lot of "file changes" events are emitted (not sure why yet) - but we only want to reload services if the config actually changed.